.travis.yml

```diff
@@ -30,6 +30,13 @@ notifications:
 os:
   - linux
 
+addons:
+    apt:
+        packages:
+            - xvfb
+            - xauth
+            - libgtk-3-0
```

Comment:
It looks like you are trying to add these dependencies to Travis?

We actually don’t run the auto merge on Travis; we run it on GH Actions.

So you would need to install these dependencies in the GH Actions workflow file here: https://github.com/JuliaRegistries/General/blob/master/.github/workflows/automerge.yml